vmck/middleware.py

```diff
@@ -0,0 +1,36 @@
+import jwt
+
+from django.http import JsonResponse
+
+from vmck import settings
+
+
+def jwt_authentication_middleware(get_response):
+    def middleware(request):
+        if request.path in settings.UNAUTHENTICATED_PATHS or settings.DEBUG:
```

Comment:
I don't think `DEBUG = True` should mean "skip authorization", is it really necessary?